tests/chainer_tests/functions_tests/loss_tests/test_decov.py

```diff
@@ -9,14 +9,13 @@
 from chainer import gradient_check
 from chainer import testing
 from chainer.testing import attr
-from chainer.testing import condition
 
 
-def _deconv(h):
+def _deconv(h, dtype):
     h = cuda.to_cpu(h)
     h_mean = h.mean(axis=0)
     N, M = h.shape
-    loss_expect = numpy.zeros((M, M), dtype=numpy.float32)
+    loss_expect = numpy.zeros((M, M), dtype=dtype)
```

Comment:
How about using `h.dtype` instead of passing `dtype`?

Reply:
Fixed!